docs/ref/contrib/syndication.txt

```diff
@@ -53,7 +53,8 @@ a feed of the latest five news items::
 
     from django.contrib.syndication.views import Feed
     from django.urls import reverse
-    from policebeat.models import NewsItem
+
+    from .models import NewsItem
```

Comment:
Leave alone. The next example implies this file is `myproject.feeds.py` rather than in the policebeat app.

Reply:
ok